app/workers/send_mail_worker.rb

```diff
@@ -0,0 +1,7 @@
+class SendMailWorker
+  include Sidekiq::Worker
+
+  def perform(rent_id)
+    ModelGeneratorMailer.new_rent_notification(rent_id).deliver
```

Comment:
You don't need a worker for this. The mailer is able to send an async mail using `deliver_later`